docs/resources/dell_idrac_attributes.md

```diff
@@ -193,4 +193,27 @@ Optional:
 - `ssl_insecure` (Boolean) This field indicates whether the SSL/TLS certificate must be verified or not
 - `user` (String) User name for login
 
+## Import
 
+Import is supported using the following syntax:
+
+```shell
+/*
+Copyright (c) 2024 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+terraform import redfish_dell_idrac_attributes.idrac '{"username":"<user>","password":"<password>","endpoint":"<endpoint>","ssl_insecure":<true/false>,"id":"<id>","attributes":{"Users.2.UserName":""}}'
```

Comment:
1. dell idrac attributes is unique on the array. It does not need an "id" to be imported.
2. "attributes" does not need to be a map. The value is not being used for import. Looks like it could have been a list of strings (keys).
3. What if the user wants to import all attributes? Surely the "attributes" field should be optional (and such an example documented)?

Reply:
address all the comments